src/pages/ProfilePage/ProfilePage.js

```diff
@@ -70,6 +70,8 @@ const ProfilePage = ({
   const [resumeLink, setResumeLink] = useState(null);
   const [openForm, setOpenForm] = useState(false);
   const [openSidebar, setOpenSidebar] = useState(false);
+  const [isOpenDrawer1, setOpenDrawer1] = React.useState(false);
+  const [isOpenDrawer2, setOpenDrawer2] = React.useState(false);
```

Comment:
Oof...so it's always a bad sign when you start having to name variables with numeric variants, like `isOpenDrawer1` and `isOpenDrawer2`. Usually that means you should refactor your code in some way.

In this instance, I'd recommend replacing these variables with a single variable, `sidebarDepth`, which is a number indicating the depth of the sidebar, and then replacing the `setOpen...` functions with `setSidebarDepth`. One benefit of this is you can actually replace _3_ variables, since a sidebarDepth of `0` would naturally correspond to closed, so you can also replace `openSidebar` as well.